asciidoc-confluence-publisher-doc/etc/docs/00-index/07-listings.adoc

```diff
@@ -2,6 +2,8 @@
 
 AsciiDoc literal blocks, listing blocks and code blocks are converted to formatted blocks in Confluence.
 
+A title can be set for each type of block. The title is rendered as the title of confluence panel.
```

Comment:
We use to write `Confluence` (with uppercase start). And maybe you could add an `the` before it(`of the Confluence panel.`)